src/Kunstmaan/AdminListBundle/Resources/views/AdminListTwigExtension/sure-modal.html.twig

```diff
@@ -12,7 +12,7 @@
                 </h3>
             </div>
             <form action="{{ path(adminlist.getDeleteUrlFor(item)["path"], adminlist.getDeleteUrlFor(item)[("params")] ) }}" method="POST">
-                <input type="hidden" name="token" value="{{ csrf_token('delete-'~adminlist.configurator.entityName|slugify) }}"/>
+                <input type="hidden" name="token" value="{{ csrf_token('node-delete') }}"/>
```

Comment:
This is not correct as it breaks the other places where an adminlist delete modal is used as this modal is also used in general (non-node related) adminlists.

Let me see if we can switch this in case of a node delete to a generic name